src/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Login</title>
+    <link rel="stylesheet" href="../../node_modules/bootstrap/dist/css/bootstrap.min.css">
+    <link rel="stylesheet" href="../../src/scss/styles.scss">
+</head>
+<body>
+    <div class="container">
+        <div class="row justify-content-center mt-5">
+          <div class="col-md-6">
+            <div class="card bg-dark text-white mt-4">
+              <div class="card-body text-center">
+                <form action="/profile" method="post">
```

Comment:
The value of the form's action attribute seems not to be working. It is returning error 405 meaning Method not allowed. This could be an issue with live-server not allowing a POST method for form action. 

Solution is to update the line to `<form action="./profile" method="get">`

Reply:
Yes, I can see what you mean. I have changed this now.